src/libLookup/Lookup.cpp

```diff
@@ -393,13 +391,6 @@ vector<unsigned char> Lookup::ComposeGetTxBlockMessage(uint64_t lowBlockNum,
 // low and high denote the range of blocknumbers being requested(inclusive).
```

Comment:
I think you can remove the comments too

Reply:
I think this comments is quite helpful, from it i know the the meaning of LowBlockNumber and HighBlockNumber.